src/dmd/attrib.d

```diff
@@ -1,4 +1,17 @@
 /**
+ * Defines declarations of various 'attributes'.
+ * The term attribute is used very loosly, since it includes:
```

Comment:
Put an empty line (well, with a `*`) between line 2 and 3 to make this part of the description.
Also I wouldn't say it is not used loosely, just that your expectation of what an attribute is might differ.
It has more to do with things applying to a larger scope than a single declaration, I'd say.